constants.js

```diff
@@ -26,3 +26,14 @@ export const FAQCategory = Object.freeze({
   MISC: 'Miscellaneous',
 });
 export const SPONSORSHIP = 'SPONSORSHIP';
+export const HACKATHON_NAVBAR = {
+  intro: 'Intro',
+  faq: 'FAQ',
+  spocos: 'Sponsors',
+  FeatureFlags: 'Feature Flags',
+};
+export const LIVESITE_NAVBAR = {
+  announcements: 'Anouncements',
+  quicklinks: 'Quicklinks',
+  schedule: 'Schedule',
```

Comment:
hmm we might need a FAQ and sponsors (for the category prizes as well as the logos) tab too unless yall are just gonna reroute it from the respective hackathon's FAQ/sponsors and manually add the category prizes

Reply:
Yeah I think we will be doing something like grabbing it from the active hackathon. Maybe we should add a settings page in the livesite area to modify which is the current 'active' hackathon?